setup.py

```diff
@@ -34,4 +34,5 @@
     zip_safe=False,
     py_modules=["Qt"],
     classifiers=classifiers,
+    setup_requires=['nose>=1.0']
```

Comment:
This line warrants its own PR, documentation and tests I think. Could we remove this for now, and implement it fully later on?